package-lock.json

```diff
@@ -4704,7 +4704,7 @@
 					}
 				},
 				"prettier": {
-					"version": "npm:prettier@2.2.1-beta-1",
+					"version": "npm:wp-prettier@2.2.1-beta-1",
```

Comment:
We shouldn't push this change because it's not pointing to the proper version. I quite often bump into this issue when installing dependencies, I usually fix it by removing the `node_modules` folder, run `npm run build:packages --prefix gutenberg` and finally run `npm install`.